apiserver/common/modelmanagerinterface.go

```diff
@@ -111,6 +112,20 @@ type CloudService interface {
 	WatchCloud(ctx context.Context, name string) (watcher.NotifyWatcher, error)
 }
 
+// MachineService defines the methods that the facade assumes from the Machine
+// service.
+type MachineService interface {
+	// EnsureDeadMachine sets the provided machine's life status to Dead.
+	// No error is returned if the provided machine doesn't exist, just nothing
+	// gets updated.
```

Comment:
Really? That's a terrible API.

Reply:
I agree, we can rework this when we switch back to machines.